hyprshot

```diff
@@ -297,5 +306,5 @@ args $0 "$@"
 
 SAVE_FULLPATH="$SAVEDIR/$FILENAME"
 [ $CLIPBOARD -eq 0 ] && Print "Saving in: %s\n" "$SAVE_FULLPATH"
-begin_grab $OPTION
+begin_grab $OPTION & checkRunning
 killHyprpicker
```

Comment:
Why does this call the killHyprpicker function at the end when checkRunning already serves that purpose?

Reply:
Great point actually